### ISSUE TITLE
[Bug] link to phpmyadmin not works

### ISSUE THREAD

**Issue by mix5003** [hestiacp/hestiacp#2843]
### Describe the bug

link phpmyadmin in https://[ip]:8083/list/db/ not works correctly (it link to http://[ip]/phpmyadmin/) 
it show 404 not found 
![image](https://user-images.githubusercontent.com/1129524/184820600-7dbea566-b458-4339-965b-713c8abd61dc.png)

### Tell us how to replicate the bug

1. install hestia 1.6.7 on ubuntu 22.04 
2. create and login to test user
3. go to database page
4. click phpmyadmin button
5. it go to http://[ip]/phpmyadmin/ but it show 404

but it can use http://[somedomain]/phpmyadmin/ normally

### Which components are affected by this bug?

Control Panel Web Interface

### Hestia Control Panel Version

1.6.7

### Operating system

Ubuntu 22.04

**Comment by ScIT-Raphael** [comment]
https://docs.hestiacp.com/admin_docs/database.html#why-i-can-t-use-http-ip-phpmyadmin

**Comment by mix5003** [comment]
if it can not use ip to access phpmyadmin by default, what the point of phpmyadmin button in db page?

**Comment by ScIT-Raphael** [comment]
use https://hostname.domain.tld:8083 with a valid ssl certificate and it will work...

**Comment by jaapmarcus** [comment]
We should update the links in the panel it is currently using http_host

**Comment by mix5003** [comment]
i think you will fix this by #2845 . i think it make sense.
but i want to suggest you to do one more think. if you prefer domain name then i think you should add another admin url that use domain name too. (may be from FQDN)
https://github.com/hestiacp/hestiacp/blob/a9fa2e3fb5d729096a565dc2251e318450855156/install/hst-install-ubuntu.sh#L2146-L2148

something like
```
    Admin URL:  https://$servername:$port 
    Alternative  URL:  https://$ip:$port
    Username:   admin
    Password:   $displaypass
```

**Comment by jaapmarcus** [comment]
We check if hostname.domain.com resolves to the ip see 

https://github.com/hestiacp/hestiacp/blob/a9fa2e3fb5d729096a565dc2251e318450855156/install/hst-install-ubuntu.sh#L2130-L2133

But a lot of users forget to setup dns records before installing the panel
